ls8/cpu.py

```diff
@@ -63,3 +65,11 @@ def trace(self):
     def run(self):
         """Run the CPU."""
         pass
+
+    def ram_read(self, MAR):
+        if MAR not in self.memory:
+            return print('MAR not found')
+        return self.memory[MAR]
+
+    def ram_write(self, MAR, MDR):
+        return self.memory[MAR] == MDR
```

Comment:
careful with your operator here, not writing anything. also, do you really want to return anything?